library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +35,22 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    public fun Context.createShortcut() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            getSystemService(ShortcutManager::class.java)?.let {
+                val shortcut = ShortcutInfo.Builder(this, "id1")
+                    .setShortLabel("Open chucker activity")
+                    .setLongLabel("Open chucker activity")
+                    .setIcon(Icon.createWithResource(this, R.mipmap.chucker_ic_launcher_round))
+                    .setIntent(getLaunchIntent(this).setAction(Intent.ACTION_VIEW))
+                    .build()
+                val allShortcuts = it.dynamicShortcuts.toMutableList()
+                allShortcuts.add(shortcut)
+                it.dynamicShortcuts = allShortcuts
```

Comment:
This can crash. You should catch exceptions that can occur here and add a meaningful log.

Also, instead of assigning new list you should just add one shortcut using `addDynamicShortcuts()`.

Reply:
Used `addDynamicShortcuts()` replace `setDynamicShortcuts()`